enzyme/mkv.py

```diff
@@ -273,22 +276,51 @@ def fromelement(cls, element):
         :type element: :class:`~enzyme.parsers.ebml.Element`
 
         """
-        targets = element['Targets'] if 'Targets' in element else []
+        targets = Targets.fromelement(element['Targets']) if 'Targets' in element else Targets()
         simpletags = [SimpleTag.fromelement(s) for s in element if s.name == 'SimpleTag']
         return cls(targets, simpletags)
 
     def __repr__(self):
         return '<%s [targets=%r, simpletags=%r]>' % (self.__class__.__name__, self.targets, self.simpletags)
 
+class Targets(object):
+    """Object for the Targets EBML element"""
+    def __init__(self, targettypevalue=50, targettype=None, trackUIDs=[], chapterUIDs=[], attachementUIDs=[], editionUIDs=[]):
```

Comment:
Don't use empty lists as default. It won't work as you expect. See http://effbot.org/zone/default-values.htm